extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -231,15 +231,19 @@ export class LiveListManager {
   /**
    * Updates the appropriate `amp-live-list` with its updates from the server.
    *
-   * @param {!Element} liveList
+   * @param {!Element} liveList Live list or custom element that built it.
    * @return {number}
    */
   updateLiveList_(liveList) {
-    // amp-live-list elements can be appended dynamically by another
-    // component using the [dynamic-live-list] attribute.
-    const id = liveList.hasAttribute('dynamic-live-list')
-      ? liveList.getAttribute('dynamic-live-list')
-      : liveList.getAttribute('id');
+    // amp-live-list elements can be appended dynamically in the client by
+    // another component using the `other-component-id` + `dynamic-list` suffix
+    // as the ID of the amp-live-list.
+    // The fact that we know how this ID is built allows us to find the
+    // amp-live-list element in the server document. See live-story-manager.js
+    // for an example.
+    const dynamicId = liveList.id + 'dynamic-list';
```

Comment:
prefix this with `i-amphtml-`
nit: put a dash between the id and `dynamic-list`

Reply:
Done.